api/src/main/resources/workbench-api.yaml

```diff
@@ -235,6 +235,35 @@ paths:
       responses:
         204:
           description: Request Received.
+
+  "/v1/getTerraTermsOfServiceStatus":
+    get:
+      tags:
+        - profile
+      description: Retrieves if the logged in user has accepted Terra service or not
```

Comment:
this should say "the current Terra terms of service"

Reply:
Done